operator-pipeline-images/operatorcert/entrypoints/static_tests.py

```diff
@@ -35,6 +45,9 @@ def setup_argparser() -> argparse.ArgumentParser:
         help="Name of the test suite to run",
         default="operatorcert.static_tests.community",
     )
+    parser.add_argument(
+        "--skip-test", help="Ignore specific checks", default=[], action=SplitArgs
```

Comment:
Since we accept a list of test names it would make more sense to call it `--skip-tests` and perhaps specify in the help text that it's interpreted as a comma separated list of test names.

Reply:
Thanks for noticing it. Fixed.